pkg/controller/worker/machines.go

```diff
@@ -30,6 +30,8 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+const KubeconfigSecretKey = "kubeconfig"
```

Comment:
Nit: can private since it is not used outside of the package.

Reply:
it's used outside of the package in the pkg `worker_test`

Reply:
Where exactly? Can you reference it?

Reply:
https://github.com/gardener/gardener-extension-provider-kubevirt/pull/9/files#diff-e61377d85357d3c316ed34aa40a57057R334

Reply:
probably you have to click "load diff" for machine_test file as it's quite big and github hides it

Reply:
But its in the same package

Reply:
it's not, the code is inside `worker` package not `worker_test`

https://tip.golang.org/cmd/go/#hdr-Test_packages
> Test files that declare a package with the suffix "_test" will be compiled as a separate package, and then linked and run with the main test binary. 

it seems the preference of testing strategy in gardener providers is to test the whole logic without accessing private identifiers, for sure it has prons, personally I have a different liking but I am not going to break already chosen strategy